docs/guides/developer/docs/reviewing-and-merging.md

```diff
@@ -44,21 +43,19 @@ all pull requests are to ensure that:
     * [Licenses seem to be fine](license.md)
     * Licenses are added to the `NOTICES` file
 * Feature documentation is in place
-* An upgrade path exists (usually relevant for database changes)
-
-
-The exact review process heavily depends on the type, size and purpose of the pull request.
```

Comment:
Streamline: "The exact review process heavily depends on the type, size and purpose of the pull request." really just isn't of any help.

Reply:
Same here: The main purpose is to make the reviewer think about the review and not stupidly follow a list of rules. That's what automated tests are for ;-)

Reply:
Just undone changes that are not directly relevant to the proposal.